From: Julien Lavergne <julien.lavergne@gmail.com>
Date: Sat, 5 Mar 2011 00:07:05 +0100
Subject: [PATCH] Some fixes for the desktop file :

- Pass validation (xfce bug 6691)
- Allows one to open blank CD and Audio CD

This patch has been slightly updated for Xubuntu to handle po/POTFILES.skip.
---
 xfburn.desktop.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xfburn.desktop.in b/xfburn.desktop.in
index 256dc9b..833e8d4 100644
--- a/xfburn.desktop.in
+++ b/xfburn.desktop.in
@@ -7,7 +7,7 @@ _Comment=CD and DVD burning application
 Exec=xfburn
 TryExec=xfburn
 Icon=stock_xfburn
-MimeType=application/x-cd-image;
+MimeType=application/x-cd-image;x-content/audio-cdda;x-content/blank-cd;
 Terminal=false
 Categories=X-XFCE;GTK;DiscBurning;Archiving;AudioVideo;Utility;
 StartupNotify=true
